documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -115,11 +118,11 @@ Try out the following example to learn how to connect two Properties to the same
 For a new model that only has an Aspect so far:
 
 . Create two Properties by clicking twice on the Aspect's Add icon image:add.png[Add icon].
-. For each Property create a Characteristic by clicking the Add icon image:add.png[Add icon] of each Property.
+. For each Property create a Characteristic by clicking the top Add icon image:add.png[Add icon] of each Property.
```

Comment:
For each Property create a Characteristic by clicking the top "Add" icon image:add.png[Add icon] of each Property.